ccd-definition/CaseField/CaseField.json

```diff
@@ -2126,7 +2126,8 @@
     "CaseTypeID": "CIVIL",
     "ID": "CaseAccessCategory",
     "Label": "Case Access Category",
-    "FieldType": "Text",
+    "FieldType": "FixedList",
+    "FieldTypeParameter": "CaseAccessCategory",
```

Comment:
You need to define a Fixed List called "CaseAccessCategory" wih the values we use in the backend:
https://github.com/hmcts/civil-service/blob/master/src/main/java/uk/gov/hmcts/reform/civil/enums/CaseCategory.java#L8

But please ensure you check backwards compatibility as these fields are live in production and we need to ensure you can do this

Reply:
CaseAccessCategory.json been added to PR.